Formula/ripasso-cursive.rb

```diff
@@ -0,0 +1,37 @@
+class RipassoCursive < Formula
+  desc "Tui password manager compatible with 'pass', written in rust"
+  homepage "https://github.com/cortex/ripasso"
+  url "https://github.com/cortex/ripasso/archive/refs/tags/release-0.6.3.tar.gz"
+  sha256 "2f43e48c3e1064bef47c89269e4797f216b7600b850f3857cfce7e9716ef8f7d"
+  license "GPL-3.0-only"
+
+  depends_on "libxcb" => :build
+  depends_on "llvm" => :build
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "tmux" => :test
+  depends_on "gpgme"
+  depends_on "libgpg-error"
+  depends_on "nettle"
+
+  on_linux do
+    depends_on "libxcb"
+  end
+
+  def install
+    ENV["SHELL_COMPLETIONS_DIR"] = buildpath
+    system "cargo", "install", "ripasso-cursive", *std_cargo_args(path: "cursive")
+
+    man1.install "target/man-page/cursive/ripasso-cursive.1"
+  end
+
+  test do
+    system "tmux", "-u", "new-session", "-d", "-x", "80", "-y", "50", "-s", "foo"
+    system "tmux", "send", "-t", "foo", "#{bin}/ripasso-cursive", "ENTER"
```

Comment:
If you're using `tmux` to mock a terminal then you might be able to use Ruby's `pty` module. See other formulae that do
```ruby
require "pty"
```
for examples.

Reply:
So I rewrote it using `pty` and that seems to work well, thanks for the suggestion that let me drop that extra test dependency.

I'm not super happy with the code, as it's racy. If the test is run on an sufficiently slow machine the program won't have time to paint a terminal with content before the sleep 1 and the 1000 iterations of read is done. But I don't think that will be a problem in practice since I really erred on the side of caution with those numbers.